src/aws/s3.tf

```diff
@@ -0,0 +1,118 @@
+# Create S3 Bucket for static files
```

Comment:
I think everything can be done in only one S3 bucket, you have a different location storage in your config so `media` and `static` will not be mixed but in a different folder.

Reply:
I usually keep static and media in different buckets because they have a different life cycle. Media files should be versioned and backed-up whereas static files are committed in your repo so we don't care about them. WDYT?

Reply:
:ok_hand: